easybuild/easyconfigs/p/petsc4py/petsc4py-3.21.0-foss-2023b.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'petsc4py'
+version = '3.21.0'
```

We should use the one from https://github.com/easybuilders/easybuild-easyconfigs/pull/22084

```suggestion
version = '3.20.3'
```